generated/api.github.com.json

```diff
@@ -208,13 +212,33 @@
         "POST /repos/{owner}/{repo}/check-suites/{check_suite_id}/rerequest"
       ]
     },
-    "codespaces": {
```

Comment:
removes `codespaces` because it's not defined in OpenAPI schema and I'm not sure if this is the correct name